src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,51 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  checkHealth() {
+    if (this.health <= 0) {
+      this.alive = false;
```

Comment:
The assignment 'this.alive = false;' is not necessary and does not contribute to the functionality as described in the task. The alive property is not part of the requirements and should be removed.